source/GameInterface.Tests/Serialization/SerializerTests/HideoutSerializationTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GameInterface.Serialization.Impl;
+using GameInterface.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem.Party.PartyComponents;
+using TaleWorlds.CampaignSystem.Settlements;
+using Xunit;
+using System.Reflection;
+using TaleWorlds.CampaignSystem;
+
+namespace GameInterface.Tests.Serialization.SerializerTests
+{
+    public class HideoutSerializationTest
+    {
+        [Fact]
+        public void Hideout_Serialize()
+        {
+            typeof(Campaign).GetProperty("Current").SetValue(null, FormatterServices.GetUninitializedObject(typeof(Campaign)));
+            List<Hideout> allhideouts = new List<Hideout>();
+            Hideout item = new Hideout();
+            allhideouts.Add(item);
+            typeof(Campaign).GetField("_hideouts", BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public).SetValue(Campaign.Current,allhideouts);
```

Comment:
Update this to be similar to https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/taverns-serialization/source/GameInterface.Tests/Serialization/SerializerTests/CharacterSkillsSerializationTest.cs#L29